modules/reference/pages/rpk/rpk-cluster/rpk-cluster-config-set.adoc

```diff
@@ -1,4 +1,5 @@
 = rpk cluster config set
+// tag::single-source[]
 
 Set a single cluster configuration property.
```

Comment:
you can also set multiples config with `rpk cluster config set iceberg_enable=true log_retention_ms=-1` format

Reply:
@andresaristizabal Is that applicable in Cloud only or does that also work for self-hosted?

Reply:
Updated

Reply:
> And if you try to set multiple configs, do you have to use the <key>=<value> syntax or does <key1> <value1> <key2> <value2> still work?

multiple option only works with ` <key>=<value>`